README.md

```diff
@@ -37,27 +59,16 @@ tuple in a list (`[{}]`) to represent empty objects. jsn is incompatible with
 this format. While the getter (`jsn:get/2,3`) functions are generally functional;
 most other library functions are not, and may result in unpredictable behaviors.
```

Comment:
That is discussed in the encode/decode deprecation. Here it does not apply, because switching to `jsone` proplists will actually break you. the choice is pretty much to use jsonx proplists or a different format (map, eep18, etc.) in this case. Hard to make a categorical recommendation.